NetBIOS

xsedla1o opened the pull request:
Plugin allowing to capture NetBIOS names with suffixes from NBNS.

havraji6 commented on netbiosplugin.cpp:
```diff
@@ -0,0 +1,174 @@
+    }
+}
+
+const char *ipfix__template[] = {
```
this should be `ipfix_netbios_template`

xsedla1o commented:
Summary of changes since last approval:
- The branch was rebased on top of master and the merge conflicts with newly added `idpcontent` plugin were resolved.
- The field type of NB_SUFFIX was changed from `bytes` to `uint8`. This was done because the suffix is always one byte long, and therefore the plugin would never utilize the variable-length `bytes` type fully.
- A dist build error that was caused by adding a non-existent target to `Makefile.am` during resolving merge conflicts was fixed.
